ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/wholeBodyHardwareControl/AvatarAffinityInterface.java

```diff
@@ -11,19 +11,19 @@
  */
 public interface AvatarAffinityInterface
 {
-   Processor getSchedulerThreadProcessor();
+   Processor getMasterThreadProcessor();
```

Comment:
where is this used? This seems like a change that is going to propagate across _a lot_ of repos. I also preferred the name scheduler to master, since it's intended to be the scheduler thread of the barrier scheduler